recipe/run_cmake_cuda_tests.sh

```diff
@@ -22,6 +22,9 @@ cd cmake-tests/build
 #   CudaOnly.GPUDebugFlag
 #   CudaOnly.SeparateCompilationPTX
 #   CudaOnly.WithDefs
+#   CudaOnly.CUBIN
+#   CudaOnly.Fatbin
+#   CudaOnly.OptixIR
```

Comment:
Curious why these were skipped

Also do these work on Linux (just not on Windows)?

Reply:
These fail on Linux due to requiring a gpu. I asked Robert about this on the other PR: https://github.com/conda-forge/cuda-nvcc-impl-feedstock/pull/5#issuecomment-1761971124